pkg/rehearse/rehearse.go

```diff
@@ -14,13 +14,12 @@ import (
 	"golang.org/x/sync/errgroup"
 
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	utilerrors "k8s.io/apimachinery/pkg/util/errors"
 	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
-	pjapi "k8s.io/test-infra/prow/apis/prowjobs/v1"
```

Comment:
This is a good one. :)

Reply:
I was not sure if it is on purpose or not, but I think it shouldn't be like that